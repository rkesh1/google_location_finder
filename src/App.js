import React from 'react';
import RouteFinder from './RouteFinder'

const App = () => (
  <RouteFinder />
)

export default App;