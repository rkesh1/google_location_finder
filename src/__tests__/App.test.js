import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("App Component Test Case ", () => {
  let wrapper;

  /**
   * Renders a shallow version of the App component
   */
  function renderShallow() {
    wrapper = shallow(<App />);
  }

  it("App Component render", () => {
    renderShallow();
    expect(wrapper).toMatchSnapshot();
  });
});
