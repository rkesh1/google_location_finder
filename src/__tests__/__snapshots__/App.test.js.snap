// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Test Case  App Component render 1`] = `ShallowWrapper {}`;
